data.sql

```diff
@@ -91,4 +91,49 @@ update animals
 set owner_id = (select id from owners where full_name = 'Dean Winchester')
 where name in  ('Angemon','Boarmon');
 
```

Comment:
- You have done great so far :+1: :+1: But there are a few Data you missed inserting into the DB. :+1: :+1: 

> Insert the following data for vets:
Vet William Tatcher is 45 years old and graduated Apr 23rd, 2000.
Vet Maisy Smith is 26 years old and graduated Jan 17th, 2019.
Vet Stephanie Mendez is 64 years old and graduated May 4th, 1981.
Vet Jack Harkness is 38 years old and graduated Jun 8th, 2008.

- [ ] You didn't write the code to insert these data above into the DB, and it's definitely going to affect the result of your queries. I'll suggest you add the data and re-run your queries. :+1: :100:

Reply:
Hey @V-Blaze , thank you for the review, I happened to forget to add the data to the .sql file but I added it to the DB I used on PgAdmin4, so the results were okay. Thank you again for the review!